include/tb_gltf.slangh

```diff
@@ -63,13 +60,16 @@ typedef struct TB_GPU_STRUCT TbGLTFDrawData {
   uint32_t vertex_offset;
   uint32_t pad0;
   uint32_t pad1;
-}
-TbGLTFDrawData;
+});
 
 #define ALPHA_CUTOFF(m) m.sheen_alpha.w
 
+#if defined(TB_SHADER) && !defined(TB_SHADER)
```

Comment:
This will never be true... right?

Reply:
Yes good catch that's an artifact that can be destroyed